splunk-otel-android/src/main/java/com/splunk/rum/Config.java

```diff
@@ -266,6 +280,17 @@ public Builder anrDetectionEnabled(boolean enable) {
             return this;
         }
 
+        /**
+         * Configures the rate at which frame render durations are polled. Pass zero to
+         * disable polling for slow/frozen renders.
+         * @param interval - The period that should be used for polling (in ms)
+         * @return this
+         */
+        public Builder renderDurationPollingIntervalMs(int interval){
```

Comment:
Rather than using a zero as a magic number here in the config, why not have an explicit way to enable/disable this feature?

Reply:
I think there's a well established common understanding that zero time means "don't do this". I think that, in this case, the zero is way clearer than having to juggle a second config item and then having handle the cases where they conflict (eg. disabled but value specified, enabled but zero specified, etc).

I'm not completely opposed to changing it, but the zero feels simpler/cleaner to me.

Reply:
Some other thoughts... maybe use a `Duration` rather than assuming ms? Or maybe an int and a TimeUnit or some such? 

I think that the name of this method is going to be confusing to users and it will be even more confusing to have to realize that disabling this requires sending in a magic number. After all, '0' could mean "do it continuously" rather than "disable it". 

Another thought to think about: this configuration might want to be included in the config blob that we send up with the initialization span.

Reply:
You could internally still use 0 or negative value for disabled when user facing api uses `Duration` and has an explicit disable method.

Reply:
> Some other thoughts... maybe use a `Duration` rather than assuming ms? Or maybe an int and a TimeUnit or some such?

I went with milliseconds based on [this past discussion in GDI spec](https://github.com/signalfx/gdi-specification/pull/150#discussion_r748397270) and [this part of the otel spec](https://github.com/open-telemetry/opentelemetry-specification/blob/2fb3b181ae2ee928039cda3baa8acc1eea701ce8/specification/sdk-environment-variables.md#duration).
 
> I think that the name of this method is going to be confusing to users and it will be even more confusing to have to realize that disabling this requires sending in a magic number. After all, '0' could mean "do it continuously" rather than "disable it".

You're wrong 😀 but if you feel that strongly about the magic number I will concede and introduce another boolean. Do you have a recommendation for a better method name?

> Another thought to think about: this configuration might want to be included in the config blob that we send up with the initialization span.

Seems reasonable.

Reply:
> > Some other thoughts... maybe use a `Duration` rather than assuming ms? Or maybe an int and a TimeUnit or some such?
> 
> 
> 
> I went with milliseconds based on [this past discussion in GDI spec](https://github.com/signalfx/gdi-specification/pull/150#discussion_r748397270) and [this part of the otel spec](https://github.com/open-telemetry/opentelemetry-specification/blob/2fb3b181ae2ee928039cda3baa8acc1eea701ce8/specification/sdk-environment-variables.md#duration).
> 
>  

Those are both about env vars. I think having it match the programmatic versions in the otel Java SDK would make sense.

Reply:
> Some other thoughts... maybe use a `Duration` rather than assuming ms? Or maybe an int and a TimeUnit or some such?

Duration is API 26, and we require 21, so no dice there right? https://developer.android.com/reference/java/time/Duration

How is the throttling exporter `windowSize` using `Duration`?

Reply:
Oh nevermind, I derped on the desugaring and verified that it's handled. I will make it a `Duration`.